### PR TITLE
Fix manual file generation and include trailing newlines in accordance with POSIX [AP-3048]

### PR DESCRIPTION
**Pull request by sbmueller**
# Description

@swift-nav/algint-team

# API compatibility

Does this change introduce a API compatibility risk?
No, this change is only fixing non-functional issues in the project.

## API compatibility plan

If the above is "Yes", please detail the compatibility (or migration) plan:

# JIRA Reference

https://swift-nav.atlassian.net/browse/AP-3048

### CODE REVIEW

File: generator/sbpg/targets/templating.py [hunk 1]
```diff
@@ -25,6 +25,7 @@
                           variable_end_string = ')))',
                           comment_start_string = '((=',
                           comment_end_string = '=))',
+                          keep_trailing_newline=True,
```
Comment (pcrumley) [inline generator/sbpg/targets/templating.py:28]: @RReichert ☝🏻

**Comment by sbmueller** [comment]
I can't modify the Dockerfile since the "build docker image" stage always fails when installing the Python dependencies :(

**Review by pcrumley — APPROVED** [review]
:shipit:

**Review by Madhunika-atmakuri — APPROVED** [review]
LGTM. Thanks @sbmueller

**Pull request merged**